utilities/rocAL/rocAL_unittests/rocAL_unittests.cpp

```diff
@@ -405,15 +407,15 @@ int test(int test_case, int reader_type, const char *path, const char *outName,
                   << "rocalRotate" << std::endl;
         image1 = rocalRotate(handle, image0, true);
     }
-    break;
+    break;*/
```

Comment:
Instead of add multiple /*, is it better use mutli line //?

Reply:
We need to add // to every line which will show more line changes
/* */ is better to use when commenting multiple lines